go.mod

```diff
@@ -0,0 +1,26 @@
+module github.com/codacy/codacy-engine-golang-seed
+
+go 1.13
+
+require (
+	github.com/360EntSecGroup-Skylar/goreporter v0.0.0-20180902115603-df1b20f7c5d0 // indirect
+	github.com/facebookgo/inject v0.0.0-20180706035515-f23751cae28b // indirect
+	github.com/facebookgo/structtag v0.0.0-20150214074306-217e25fb9691 // indirect
+	github.com/fatih/color v1.8.0 // indirect
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b // indirect
+	github.com/josemiguelmelo/gofile v0.0.0-20200102193424-d0d563814b61
+	github.com/json-iterator/go v1.1.9 // indirect
+	github.com/kisielk/gotool v1.0.0 // indirect
+	github.com/mattn/go-colorable v0.1.4 // indirect
+	github.com/mattn/go-isatty v0.0.11 // indirect
+	github.com/mgechev/revive v1.0.0 // indirect
```

Comment:
I don't think we should already depend on a tool, can you verify why we have this dependency?

Reply:
Nice catch! Removed all unused dependencies